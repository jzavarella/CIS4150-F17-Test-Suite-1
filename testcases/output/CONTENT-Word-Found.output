B AA1 R
DONE! (1 lookups)
