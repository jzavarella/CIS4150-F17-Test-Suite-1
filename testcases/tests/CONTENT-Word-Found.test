#!/bin/sh

echo "BAR" | ./testcases/bin/cmulookup testcases/includes/full-dictionary
