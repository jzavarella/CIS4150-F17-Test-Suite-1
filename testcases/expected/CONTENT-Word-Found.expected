B AA1 R
DONE! (1 words)
